notebooks/service/service.go

```diff
@@ -52,32 +52,18 @@ func (p Page) Validate() error {
 	return nil
 }
 
-// NotebookCreate contains fields for creating notebooks.
-type NotebookCreate struct {
```

Comment:
The previous `NotebookCreate` and `NotebookUpdate` types were able to be combined into a single type, since the `notebooksd` service requires the same fields for both.